peer/brontide.go

```diff
@@ -693,7 +692,7 @@ func (p *Brontide) String() string {
 // readNextMessage reads, and returns the next message on the wire along with
 // any additional raw payload.
 func (p *Brontide) readNextMessage() (lnwire.Message, error) {
-	noiseConn, ok := p.cfg.Conn.(*brontide.Conn)
+	noiseConn, ok := p.cfg.Conn.(MessageConn)
```

Comment:
could we change the type of `p.cfg.Conn` to `MessageConn`?

Reply:
Yeah that makes sense